conversions/conversions.py

```diff
@@ -33,35 +34,44 @@ def switch_columns(self):
 
     # converts voltage to mm and then inches for as spring rates are in inches / pound
     def convert_voltage_to_in(self):
-        for i, row in self.data.iterrows():
-            self.data.loc[i, "Front Right"] = (-(row["Front Right"] * constants.LINPOT_CONVERSION_CONSTANT) +
-                                               constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
-            self.data.loc[i, "Front Left"] = (-(row["Front Left"] * constants.LINPOT_CONVERSION_CONSTANT) +
-                                              constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
-            self.data.loc[i, "Rear Right"] = (-(row["Rear Right"] * constants.LINPOT_CONVERSION_CONSTANT) +
-                                              constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
-            self.data.loc[i, "Rear Left"] = (-(row["Rear Left"] * constants.LINPOT_CONVERSION_CONSTANT) +
-                                             constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
-
+        try: 
+            for i, row in self.data.iterrows():
+                self.data.loc[i, "Front Right"] = (-(row["Front Right"] * constants.LINPOT_CONVERSION_CONSTANT) +
+                                                   constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
+                self.data.loc[i, "Front Left"] = (-(row["Front Left"] * constants.LINPOT_CONVERSION_CONSTANT) +
+                                                  constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
+                self.data.loc[i, "Rear Right"] = (-(row["Rear Right"] * constants.LINPOT_CONVERSION_CONSTANT) +
+                                                  constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
+                self.data.loc[i, "Rear Left"] = (-(row["Rear Left"] * constants.LINPOT_CONVERSION_CONSTANT) +
+                                                 constants.LINPOT_CONVERSION_OFFSET) * constants.MM_TO_IN_CONVERSION_FACTOR
+        except Exception:
+            traceback.print_exc()
+            print("Data in csv file is not right")
     # fill out method for converting voltage to mm
     def convert_voltage_to_mm(self):
         pass
 
     def clean_data(self):
         filter_instance = Filter()
-        self.data = filter_instance.butter_lowpass_filter(self.data, "Front Right", 4, 30, 2)
-        self.data = filter_instance.butter_lowpass_filter(self.data, "Front Left", 4, 30, 2)
-        self.data = filter_instance.butter_lowpass_filter(self.data, "Rear Right", 4, 30, 2)
-        self.data = filter_instance.butter_lowpass_filter(self.data, "Rear Left", 4, 30, 2)
+        try:
```

Comment:
no need for this one either, the better place to put it would be within the filter itself